ui-tests/cypress/support/pageObjects/administration/languages_Labels/Languages_LabelsPage.js

```diff
@@ -46,12 +49,14 @@ export default class Languages_LabelsPage extends Content {
 
   getLanguageTable() {
     return this.getLanguageArea()
-      .find(`[${DATA_TESTID}=list_ActiveLangTable_table]`);
+      .find(`${htmlElements.table}.ActiveLangTable__table`);
   }
 
   getDeleteLanguageByCode(code) {
     return this.getLanguageTable()
-      .find(`#ActiveLangTable-delete-${code}`);
+      .contains(code)
```

Comment:
The contains method works on the text used, which is more subject to change, do not affect the functionality and might be language dependent; try to find a different way to identify this element. Is the old _html id_ not present anymore?

Reply:
The old html id is no longer present, and there's no other identifier for each row aside from the contained text, so I'm not sure how to do this without using a `contains` in this particular case. I feel like selectors wouldn't be a good choice, since this is a dynamic table.

Reply:
If it is so, maybe you can identify the main element (the one containing the txt) in the test code, as it is part of the testing process (the code to use will be defined by the test itself), then pass it to the page object method to interact with it.
The main focus is to split what is structure, and hence defined in the page object, and what is dynamically defined, used and expected by the test.

Reply:
Done